jgrapht-io/src/main/java/org/jgrapht/io/GmlExporter.java

```diff
@@ -66,15 +71,13 @@
         /**
          * If set the exporter outputs edge weights
          */
-        EXPORT_EDGE_WEIGHTS
+        EXPORT_EDGE_WEIGHTS,
+        /**
+         * If set the exporter escapes all strings as Java strings
```

Comment:
should we specify what the alternative is? E.g. "If set the exporter escapes all strings as Java strings, alternatively, if not set, .... "